GOVERNANCE.md

```diff
@@ -0,0 +1,205 @@
+The nomination is done as a pull request to the *CODEOWNERS* file in corresponding project's git repository. The approval must be done on the pull request by project's maintainers.
+
+### Maintainers
+
+Maintainers are active contributors and participants in the projects they are maintainers of. Maintainers have the rights to accept pull requests.
+
+Maintainers are eligible to vote on Technical Committee elections. They can be nominated as members to the Technical Committee during Technical Committee elections.
+
+Maintainers have right to nominate contributors to become maintainers and vote on the maintainer promotions for the projects they are maintainers of.
+
+Individuals can be maintainers to multiple Eiffel Community Open Source Projects.
+
+## Technical Committee
+
+The Technical Committee is responsible for the oversight of the Eiffel Community Open Source Projects. Technical Committee is comprised of **X** voting members.
+
+The term for Technical Committee is **one year**.
+
+The responsibilities of the Technical Committee are
+
+* coordination of the technical direction of the prroject(s)
+* discussions, seeking consensus, and where necessary, voting on technical matters that affect multiple Eiffel Community Open Source Projects
+* serving as point of contact with regards to licensing matters
+* determination and amendment of the size, makeup, and the procedure for Technical Committee
+* establishment of the project creation and archival processes, approving and archiving Eiffel Community Open Source Projects
+* creation and amendment of community governance, election, contribution, and code of conduct guidelines
+* enforcement and reminding of code of conduct processes
+* creation and amendment of the criteria for maintainer promotion and removal
+* creation or elimination of project and community roles
+* appointing community members to interact with other open source communities
+* coordinating event participation and communications regarding the Eiffel Community Open Source Projects
+
+Eiffel Community aims to operate as a consensus based community, if any decision requires a vote to move the Eiffel Community Open Source Projects forward, the members of the Technical Committee will vote on a one vote per member basis.
+Simple majority is required for any decision to be approved.
+
+The Technical Committee meets regularly and the meetings are intended to be open to the public and can be conducted electronically, via teleconference, or in person.
+Quorum for Technical Committee meetings requires at least a majority of all voting members of the Technical Committee to be present.
+The Technical Committee  may continue to meet if quorum is not met, but will be prevented from making any decisions at the meeting.
+
+### Technical Committee Members
+
+Members of the Technical Committee are core stakeholders and/or contributors.
```

Comment:
Need to define core stakeholder, I could not find it in the document elsewhere.

Reply:
This is a leftover from Tekton governance document. We can exclude this sentence to avoid confusion as everyone who takes part in/contributes to Eiffel is stakeholder in my opinion.